Sources/ApolloCodegenLib/Frontend/CompilationResult.swift

```diff
@@ -13,6 +15,31 @@ public class CompilationResult: JavaScriptObject {
 
   lazy var schemaDocumentation: String? = self["schemaDocumentation"]
   
+  required init(_ jsValue: JSValue, bridge: JavaScriptBridge) {
+    super.init(jsValue, bridge: bridge)
+    processRootTypes()
```

Comment:
Looking through the code, and where/how the `isRootFieldType` was originally used, without piping the new root type data through the pipeline, or updating the `GraphQLCompositeType` objects later in the pipeline, this seems like a good way to ensure all of the `GraphQLCompositeType` objects have their `isRootFieldType` updated before use.